src/main/views/create-media-account.njk

```diff
@@ -3,6 +3,7 @@
 {% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
```

Comment:
E2E tests will need updating to account for the checkbox, as we create a media account as part of them

Reply:
This has now been done :)